compiler/src/yul/constants.rs

```diff
@@ -0,0 +1,57 @@
+use fe_analyzer::namespace::types::Integer;
+use maplit::hashmap;
+use std::collections::HashMap;
+use yultsur::*;
+
+pub fn numeric_min_max() -> HashMap<Integer, (yul::Expression, yul::Expression)> {
```

Comment:
I wanted to have this as an actual constant using [phf](https://docs.rs/phf/0.8.0/phf ) but that would have required `Integer` to to implement `PhfHash` as it doesn't work with the builtin `Hash` trait. But requiring to depend on `phf` across multiple crates just for this didn't feel quite right to me. I gave up and left this as a method.

Reply:
Its fine. Could we get some docs on this?